### PR TITLE
AD-352: Implement Sourcepoint CMP code snippets

### PR DESCRIPTION
**Pull request by iolayemi**

### CODE REVIEW

**Review by AmandaVaughn — APPROVED** [review]
Looks great to me, follows the implementation instructions for Freestar Sourcepoint CMP found here: https://help.freestar.com/help/policy-freestar-hosted-cmp

File: html/index.html [hunk 1]
```diff
@@ -1205,4 +1205,7 @@
       gtag('config', 'G-WMZZXH53QH');
     </script>
   </body>
+  <footer>
+    <button id="pmLink">Privacy Manager</button>
```
Comment (VictorM-jetnet) [inline html/index.html:1209]: @iolayemi , just out of curiosity, are there any handlers assigned to this button? how does it work when the user click on it? is it managed by some snippet embedded?

Comment (iolayemi) [reply html/index.html:1209]: That is the question I asked Dan during standup. 
My understanding is that there is a library embedded which is responsible for targeting this button and styling

Comment (VictorM-jetnet) [reply html/index.html:1209]: I see. ok, sounds good, thank you Olayemi

**Pull request merged**